apps/fz_http/lib/fz_http/events.ex

```diff
@@ -5,38 +5,50 @@ defmodule FzHttp.Events do
 
   alias FzHttp.{Devices, Rules, Users}
 
+  require Logger
+
   # set_config is used because devices need to be re-evaluated in case a
   # device is added to a User that's not active.
-  def update_device(device) do
-    GenServer.call(wall_pid(), {:add_device, Devices.setting_projection(device)})
-    GenServer.call(vpn_pid(), {:set_config, Devices.to_peer_list()})
+  def add(subject, device) when subject == "devices" do
+    with :ok <- GenServer.call(wall_pid(), {:add_device, Devices.setting_projection(device)}),
+         :ok <- GenServer.call(vpn_pid(), {:set_config, Devices.to_peer_list()}) do
+      :ok
+    else
+      # XXX: propagate error to ui in 0.5.1
```

Comment:
I snuck 0.5.1 (and 0.5.2) out on ya, sorry. Maybe remove the version?

Reply:
Yeah #867 which adds the notifications view for these errors takes this out.